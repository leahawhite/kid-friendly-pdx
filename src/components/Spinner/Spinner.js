import React, { Component } from 'react'
import './Spinner.css'

export default class Pinwheel extends Component {
  render() {
    return (
      <div className="spinner">
        <img src={require('../../images/pinwheel.svg')} alt="pinwheel spinner" />
      </div>
    )
  }
}