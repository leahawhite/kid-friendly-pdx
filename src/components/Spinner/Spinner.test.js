import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Spinner from './Spinner';

describe.only('Spinner component', () => {
  
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Spinner />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders UI as expected', () => {
    const wrapper = shallow(<Spinner />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})