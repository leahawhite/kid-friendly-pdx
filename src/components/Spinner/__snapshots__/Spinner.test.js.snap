// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component renders UI as expected 1`] = `
<div
  className="spinner"
>
  <img
    alt="pinwheel spinner"
    src={
      Object {
        "ReactComponent": Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        },
        "__esModule": true,
        "default": "pinwheel.svg",
      }
    }
  />
</div>
`;
